arabic/salamoAlikom.md

```diff
@@ -1,4 +1,4 @@
-...السلام عليكم و رحمة الله و بركاته...
```

Comment:
It looks to me like whatever change you made was just whitespace, so if you break up the long line, I'll be happy to merge this.